{"date": "August 1, 2019", "cve": "CVE-2019-14232", "summary": "Denial-of-service possibility in django.utils.text.Truncator", "link": "https://www.djangoproject.com/weblog/2019/aug/01/security-releases/", "versions": [["1.11", "1.11.23"], ["2.1", "2.1.11"], ["2.2", "2.2.4"]]}